package.json

```diff
@@ -4,20 +4,39 @@
   "license": "Apache-2.0",
   "private": true,
   "engines": {
-    "node": ">=12"
+    "node": ">=20.0.0"
   },
   "workspaces": [
     "pkg/*"
   ],
   "dependencies": {
     "@rancher/components": "0.2.1-alpha.0",
-    "@rancher/shell": "^2.0.1",
-    "@types/lodash": "4.14.184",
-    "core-js": "3.21.1"
+    "@rancher/shell": "^3.0.0-rc.8",
+    "cache-loader": "^4.1.0",
+    "node-polyfill-webpack-plugin": "^3.0.0"
```

Comment:
I cannot find the packages (node-polyfill and cache-loader) being added to the webpack config